cql3/restrictions/statement_restrictions.cc

```diff
@@ -1504,6 +1518,12 @@ bool is_satisfied_by(const expression& restr, column_value_eval_bag bag) {
 
 } // anonymous namespace
 
+expression make_conjunction(expression a, expression b) {
```

Comment:
Should this be in some other place?

Reply:
Not sure what you mean -- other file, other namespace, other commit?  It's been in this file and namespace since birth, though previously it's been in an earlier commit.

Reply:
There was an earlier commit that introduced the functions working on the data structures, which would be a better place for this. Right now, if I'm not mistaken, it's in the commit that integrates everything.

Reply:
A previous version put make_conjunction in the same commit that introduced conjunction and expression.  That commit didn't have any callers to make_conjunction, though.  I thought it better to introduce just the data structures first and add make_conjunction together with the code that uses it.

Reply:
Then again, my second commit now introduces a bunch of functions on expression that aren't called anywhere until the third commit. :man_shrugging: I don't know what's best...

Reply:
Moved back to the second commit, to minimize surprises for the reviewer.